assets/src/dashboard/app/views/editorSettings/publisherLogo/popoverLogoContextMenu.js

```diff
@@ -50,12 +52,16 @@ function PopoverLogoContextMenu({
   );
 
   const handleFocusOut = useCallback(() => {
-    if (contextMenuId.value === activePublisherLogo) {
+    if (contextMenuId.value === publisherLogo.id) {
```

Comment:
Context: This was a *super weird bug*. Big thanks to @embarks and @BrittanyIRL for pairing on this with me.

**Part 1**
When clicking an item in the menu, the focusout event would get triggered for the publisher logo that wasn't being interacted with. This would then close the context menu because we were comparing `contextMenuId` and `activePublisherLogo` (which were the same in the parent component) but didn't match the `publisherLogo.id`

The menu would close before the click event handler would fire. Changing this to `publisherLogo.id` allowed click events to be called once again.

Reply:
A new bug appeared. Continued in **Part 2**